client/signup/controller.js

```diff
@@ -83,6 +91,7 @@ export default {
 			React.createElement( SignupComponent, {
 				path: context.path,
 				refParameter,
+				jetpackRedirect,
```

Comment:
@drewblaisdell what do you think about just passing down context.query here instead?